tests/test_domain.py

```diff
@@ -184,5 +184,5 @@ def test_domain_to_yaml():
   utter_greet:
   - text: hey there!"""
     domain = TemplateDomain.load_from_yaml(test_yaml)
-    assert test_yaml.strip() == domain.as_yaml().strip()
+    assert domain.as_yaml().strip().endswith(test_yaml.strip())
```

Comment:
Using Python 3 the document will start with a pragma when it won't with Python 2, so `==` won't work anymore.

Reply:
do you mind adding this as a comment so we keep this in mind?